install_files/ansible-base/roles/app/tasks/mod_pagespeed.yml

```diff
@@ -0,0 +1,61 @@
+---
+# TODO remove when mod_pagespeed is an fpf deb package
```

Comment:
Yeah, the idea was security since you guys already maintain other packages. Not trying to force more work on y'all. :D